src/app/cases/case/case.component.ts

```diff
@@ -10,15 +10,35 @@ import { CaseOrigin } from '../../_models/caseOrigin';
 // case routing for tabs
 const caseLinks = (caseId: string): EntityLink[] => {
   return [
-    { link: `/cases/case/${caseId}/details`, title: 'strings.entityCase' },
-    { link: `/cases/case/${caseId}/person`, title: 'captions.View.cases.person' },
+    { link: `/cases/case/${caseId}/details`, title: 'strings.entityCase', showFormActions: true },
+    {
+      link: `/cases/case/${caseId}/person`,
+      title: 'captions.View.cases.person',
+      showFormActions: true,
+    },
     { link: `/cases/case/${caseId}/hospitalization`, title: 'captions.CaseHospitalization' },
-    { link: `/cases/case/${caseId}/port-health`, title: 'captions.CaseData.portHealthInfo' },
-    { link: `/cases/case/${caseId}/symptoms`, title: 'captions.Symptoms' },
-    { link: `/cases/case/${caseId}/epidemiological-data`, title: 'captions.EpiData' },
+    {
+      link: `/cases/case/${caseId}/port-health`,
+      title: 'captions.CaseData.portHealthInfo',
+      showFormActions: true,
+    },
+    { link: `/cases/case/${caseId}/symptoms`, title: 'captions.Symptoms', showFormActions: true },
+    {
+      link: `/cases/case/${caseId}/epidemiological-data`,
+      title: 'captions.EpiData',
+      showFormActions: true,
+    },
     { link: `/cases/case/${caseId}/therapy`, title: 'captions.CaseData.therapy' },
-    { link: `/cases/case/${caseId}/follow-up`, title: 'captions.caseFollowupVisitsView' },
-    { link: `/cases/case/${caseId}/clinical-course`, title: 'captions.View.cases.clinicalcourse' },
+    {
+      link: `/cases/case/${caseId}/follow-up`,
+      title: 'captions.caseFollowupVisitsView',
+      showFormActions: true,
+    },
+    {
+      link: `/cases/case/${caseId}/clinical-course`,
+      title: 'captions.View.cases.clinicalcourse',
+      showFormActions: true,
+    },
```

Comment:
wouldn;t have been easier to set the showFormActions to false and have it default true ? Because you have it only in one pace on false and 90% of cases are true

Reply:
Like this it makes more sense as the word for it is showFormActions. And I don't care if it's not set or it's false.